go/api/options/zintercard_options.go

```diff
@@ -0,0 +1,31 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+package options
+
+import (
+	"github.com/valkey-io/valkey-glide/go/glide/utils"
```

Comment:
```suggestion
	"github.com/valkey-io/valkey-glide/go/utils"
```

Please remove all `glide` in go imports going forward, after the merge of #3062